ethstorage/prover/README.md

```diff
@@ -27,4 +27,50 @@ Go-rapidsnark also provides the go version of the witness calculator, so that ou
 
 We also need to investigate the gnark to decide whether we want to choose it or rapidsnark, but at least we have a better alternative than snarkjs
 
-source: https://github.com/ethstorage/go-ethstorage/pull/14#issuecomment-1590816080
\ No newline at end of file
+source: https://github.com/ethstorage/go-ethstorage/pull/14#issuecomment-1590816080
+
+
+
+### Performance updates
+
+1. Tests on MacOS amd64 show no performance gain of go-rapidsnark.
+
+ Implementation  | Proof per sample (mode 1) | Proof for 2 samples (mode 2) 
```

Comment:
Should we update the latest test result here?

Reply:
Test result with go-rapidsnark/prover@v0.0.11

platform|Proof for 2 samples (mode 2)| go version
-- | -- | -- 
MBP-AMD |26.8s | go1.22.3 darwin/amd64
AX-101| 15.6s |  go1.20.10 linux/amd64